pornhub/core.py

```diff
@@ -59,7 +60,7 @@ def isStar(url):
     Validate pornstar's page
     www.pornhub.com/pornstar/SOMENAME
     """
-    return True if PORNSTAR_URL in url else False
+    return True if PORNSTAR_URL or MODEL_URL in url else False
```

Comment:
This does not look right. I guess you meant `(PORNSTAR_URL in url) or (MODEL_URL in url)`?